test/Tizen.NUI.WindowSystem.InputGenerator/Tizen.NUI.WindowSystem.InputGenerator.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
```

Comment:
very good to add sample APP!
could you add copy right statement?
thank you.

Reply:
Done :smile: